[REFACTOR] 코드리뷰

ujiiin opened the pull request:
## ✒️ 관련 이슈번호

## 🔑 Key Changes

1. 내용
    - API 명세서에 있는 담당에 맞춰 코드 리뷰 진행하고, 코드 리뷰 후 명세서에 진행 상황 업데이트 해주세요
    - 검수자는 dev에서 pull받아 검수 후 PR에 comment 달아주세요
    - comment에 모두 답변을 달아야 머지가 가능합니다. 담당자는 코드 리뷰를 반영한 후 comment에 적절한 답변을 달아주세요
    - 코드 수정 내용이 많을 경우 새 이슈 및 브랜치를 파서 작업해주세요
    - dev에 push하거나, 브랜치를 merge하기 전 로컬에서 오류 없이 잘 돌아가는지 꼭 확인해주세요
    - 막아져있긴 하겠지만 main에서 push하거나 PR을 merge하지 않도록 조심해주세요. 
    - 항상 현재 내가 어느 브랜치에 있는지 꼭 확인해주세요

## 📸 Screenshot

hyoinkang committed:
Merge pull request #45 from UMC-ShowHoo/feature/#39

[FEAT] 예매 내역 상태 변경 API 구현

hyoinkang committed:
Merge pull request #50 from UMC-ShowHoo/feature/#46

[FEAT] 관람자/공연 찜 기능 구현

qormoon committed:
feat: 공연장 사진 등록 api 구현

qormoon committed:
refactor: 공연장 등록 시 공연장 사진 String으로 받아 저장 #53

qormoon committed:
refactor: 공연장 등록 시 계좌번호 추가 #53

qormoon committed:
feat: 성수기 대관료 엔티티 생성 #53

dldusgh318 committed:
[FIX] dev pull (#34)

dldusgh318 committed:
[Feat] 공연자/공연장 - 대관 양식, 대관 서류 업로드/다운로드 API 구현 (#34)

dldusgh318 committed:
[FIX] dev pull (#34)

dldusgh318 committed:
Merge pull request #54 from UMC-ShowHoo/feature/#34

[FEAT] 공연자/공연장 - 대관 양식, 대관 서류 업로드/다운로드 API 구현 (#34)

qormoon committed:
feat: 공연장 등록 시 성수기 대관료 저장할 수 있게 추가 #53

qormoon committed:
feat: 휴무일 엔티티 생성 #53

qormoon committed:
feat: 공연장 등록에 휴무일 추가 #53

qormoon committed:
fix: db 중복 저장 오류 해결 #53

hyesuhan committed:
feat: 대관 수락 및 거절

hyesuhan committed:
Revert "feat 대관 api"

This reverts commit 443dc08b120c63033575639c913c5498977a4ddd.

# Conflicts:
#	src/main/java/umc/ShowHoo/web/spaceApply/controller/SpaceApplyController.java
#	src/main/java/umc/ShowHoo/web/spaceApply/repository/SpaceApplyRepository.java
#	src/main/java/umc/ShowHoo/web/spaceApply/service/SpaceApplyService.java

hyesuhan committed:
Merge remote-tracking branch 'origin/dev' into dev

# Conflicts:
#	.github/workflows/gradle.yml
#	src/main/java/umc/ShowHoo/web/audience/entity/Audience.java
#	src/main/java/umc/ShowHoo/web/book/controller/BookController.java
#	src/main/java/umc/ShowHoo/web/book/dto/BookRequestDTO.java
#	src/main/java/umc/ShowHoo/web/book/dto/BookResponseDTO.java
#	src/main/java/umc/ShowHoo/web/book/service/BookCommandService.java
#	src/main/java/umc/ShowHoo/web/space/entity/Space.java
#	src/main/resources/application-prod.yml

qormoon committed:
refeactor: 가격 조회 api 수정 #53

qormoon committed:
feat: 성수기일 때 가격 변화 #53

dldusgh318 committed:
[MODIFY] Book 테이블 결제금액 추가 (#34)

dldusgh318 committed:
Merge pull request #55 from UMC-ShowHoo/feature/#34

[MODIFY] Book 테이블 결제금액 추가 (#34)

hyoinkang committed:
feature : 공연 게시글 조회 API 시그니처 작성

hyoinkang committed:
Merge branch 'dev' of https://github.com/UMC-ShowHoo/ShowHoo-Server into feature/#51

qormoon committed:
feat: 대관신청하기 결제하기 api 구현 #53

qormoon committed:
chore: 대관신청하기 api 설명 추가 #53

hyoinkang committed:
feature : 공연 게시글 조회 API 서비스 로직 작성

qormoon committed:
Merge pull request #57 from UMC-ShowHoo/feature/#53

[FEAT] 공연장 등록, 공연장 세부정보 조회, 대관신청 추가 구현

hyoinkang committed:
feature : 공연 게시글 조회 API + 찜 여부 조회 로직 작성

hyoinkang committed:
Merge branch 'dev' of https://github.com/UMC-ShowHoo/ShowHoo-Server into feature/#51

dldusgh318 committed:
[FEAT] 공연자 - 공연 준비: 예매자 관리 API 구현 (#56)

qormoon committed:
feat: 공연자 프로필 이미지 업로드 api 구현 #58

qormoon committed:
refactor: 공연자 프로필 등록 api 사진 url로 받게 수정 #58

qormoon committed:
refactor: 프로필 이미지 수정 시 추가한 이미지 url 반환 #58

ujiiin committed:
feat: notification 엔티티 생성 #48

ujiiin committed:
feat: notification member 매핑 #48

ujiiin committed:
feat: performer, spaceUser, audience 에러핸들러 및 repo 추가 매핑 #48

ujiiin committed:
feat: member, performerProfile 핸들러 추가 및 errorStatus 수정 #48

ujiiin committed:
feat: 알림 생성, 삭제, 조회 API 구현 #48

qormoon committed:
feat: 리뷰 이미지 업로드 api 구현 #58

ujiiin committed:
feat: 대관 신청시 공연장에게 알림 #48

qormoon committed:
refactor: 리뷰 작성하기 api 사진 부분 수정 #58

ujiiin committed:
refactor: 대관 신청시 공연장에게 알림 #48

qormoon committed:
Merge pull request #64 from UMC-ShowHoo/feature/#58

[MODIFY] 리뷰, 공연자 프로필 사진 관련 api 추가, 수정

ujiiin committed:
feat: notification handler 작성 #48

ujiiin committed:
feat: 공연장 리뷰 작성시 공연장에게 알림 #48

ujiiin committed:
feat: 공연장 리뷰 댓글 작성시 performer에게 알림 #48

ujiiin committed:
feat: 공연 수락시 관람자에게 알림 #48

ujiiin committed:
Merge branch 'dev' into feature/#48

ujiiin committed:
Merge pull request #65 from UMC-ShowHoo/feature/#48

[FEAT] 알림 API 구현 및 공연장, 관람자 알림 구현

hyoinkang committed:
feature : 카카오 연결 끊기 기능 구현

dldusgh318 committed:
[FIX] 공연자 - 공연준비 : 공연 정보 등록 API 멀티파트 수정 (#66)

dldusgh318 committed:
[FIX] 공연자 - 공연준비 페이지 swagger responses 수정 (#66)

dldusgh318 committed:
[FIX] 공연자 - 공연준비 예매자 관리 pull (#66)

dldusgh318 committed:
[FIX] 공연자/공연장 - 공연준비: 큐시트 작성 페이지 swagger responses 수정(#66)

dldusgh318 committed:
[MODIFY] 공연자 공연준비: shows 테이블의 공연 설명 삭제 (#68)

dldusgh318 committed:
[MODIFY] 공연자 공연준비: 공연 설명 테이블(ShowsDescription) 설계(#68)

dldusgh318 committed:
[MODIFY] 공연자 공연준비: 공연 설명 API 구현(#68)

hyesuhan committed:
fix: 공연 정보 작성 및 요청 사항

hyoinkang committed:
Merge pull request #59 from UMC-ShowHoo/feature/#51

[FEAT] 관람자 / 공연 게시글 조회 API 구현

hyoinkang committed:
Merge pull request #63 from UMC-ShowHoo/feature/#56

[FEAT] 공연자 - 공연 준비: 예매자 관리 API 구현 (#56)

hyoinkang committed:
Merge pull request #67 from UMC-ShowHoo/feature/#66

[MODIFY] 공연자 - 공연준비 : 공연 정보 등록 API 수정

hyoinkang committed:
Merge pull request #69 from UMC-ShowHoo/feature/#68

[MODIFY] 공연자 공연 준비: 공연설명 수정 (#68)

hyoinkang committed:
Merge branch 'dev' of https://github.com/UMC-ShowHoo/ShowHoo-Server into feature/#62

dldusgh318 committed:
[FIX] dev 병합 오류 수정중 (#68)

dldusgh318 committed:
Merge branch 'dev' of https://github.com/UMC-ShowHoo/ShowHoo-Server into feature/#68

hyoinkang committed:
fix : 에러 해결

hyoinkang committed:
Merge pull request #72 from UMC-ShowHoo/feature/#60

fix : 에러 해결

hyoinkang committed:
Merge branch 'dev' of https://github.com/UMC-ShowHoo/ShowHoo-Server into feature/#62

ujiiin committed:
feat: 공연자 마이페이지 조회 API 구현 #31

ujiiin committed:
feat: 찜한 공연장 조회 API 구현 #31

ujiiin committed:
feat: 공연장 유저 마이페이지 API 구현 #31

ujiiin committed:
Merge pull request #74 from UMC-ShowHoo/feature/#31

[FEAT] 공연자, 공연장 마이페이지 조회

hyoinkang committed:
modify : 티켓 가격 정보 추가

hyoinkang committed:
feature : 공연 티켓 매진 validator 구현

hyoinkang committed:
modify : Shows 엔티티 연관관계 수정 (Performer -> PerformerProfile) 및 관련 오류 해결

hyoinkang committed:
Merge pull request #77 from UMC-ShowHoo/modify/#73

[MODIFY] 관람자 / 공연 예매 API 수정

hyoinkang committed:
feature : 예매 상태 업데이트 스케줄러 작성

ujiiin committed:
ci: application-prod 8081 포트 설정 #31

ujiiin committed:
refactor: capacity값이 없을 경우 0으로 처리 #71

ujiiin committed:
refactor: review가 작성될 때 grade 업데이트 #71

ujiiin committed:
Merge pull request #80 from UMC-ShowHoo/refactor/#71

[REFACTOR] 공연장 전체 조회 API grade값 수정

ujiiin committed:
feat: 공연자 알림 - 공연 취소 되었을 때, 대관 취소 되었을 때 추가 #71

ujiiin committed:
fix: merge 및 에러 해결 #36

ujiiin committed:
Merge pull request #70 from UMC-ShowHoo/feature#36

[FEAT] 대관 거절 수락 API 구현

ujiiin committed:
Revert "대관 거절 수락 관련 API"

ujiiin committed:
Merge pull request #82 from UMC-ShowHoo/revert-70-feature#36

Revert "대관 거절 수락 관련 API"

ujiiin committed:
feat: 대관 승인 API 복구 #83

ujiiin committed:
Merge pull request #84 from UMC-ShowHoo/feature/#83

[FEAT] 대관 승인 API 복구 #83

dldusgh318 committed:
[FIX] dev pull해옴 (#68)

ujiiin committed:
Merge branch 'dev' of https://github.com/UMC-ShowHoo/ShowHoo-Server into feature/#81

qormoon committed:
Merge pull request #78 from UMC-ShowHoo/modify/#75

[MODIFY] Shows 엔티티 연관관계 수정

dldusgh318 committed:
[FIX] 멀티파트, 글 동시 post 할 때 생기는 오류 수정 (#68)

qormoon committed:
Merge pull request #79 from UMC-ShowHoo/feature/#76

[FEAT] 공연 예매 상태 업데이트 스케줄러 작성

dldusgh318 committed:
[FIX] dev pull (#68)

dldusgh318 committed:
Merge pull request #85 from UMC-ShowHoo/feature/#68

[FIX] 멀티파트, 글 동시 post 할 때 생기는 오류 수정 (#68)

ujiiin committed:
feat: 대관 승인 알림 #83

ujiiin committed:
Merge branch 'dev' of https://github.com/UMC-ShowHoo/ShowHoo-Server into feature/#81

ujiiin committed:
feat: 알림에서 perfomerProfile name 가져오도록 변경 #83

hyoinkang committed:
feature : 카카오계정과 함께 로그아웃

ujiiin committed:
Merge pull request #86 from UMC-ShowHoo/feature/#81

[FEAT] 공연자 알림 구현 #81

hyoinkang committed:
Merge branch 'dev' of https://github.com/UMC-ShowHoo/ShowHoo-Server into feature/#61

ujiiin committed:
feat: 알림 개수 조회 API 구현 #87

hyoinkang committed:
Merge pull request #88 from UMC-ShowHoo/feature/#61

[FEAT] 카카오 계정과 함께 로그아웃

ujiiin committed:
Merge remote-tracking branch 'origin/dev' into dev

hyoinkang committed:
Merge branch 'dev' into feature/#62

hyoinkang committed:
Merge pull request #90 from UMC-ShowHoo/feature/#62

[FEAT] 카카오 연결 끊기

ujiiin committed:
feat: 공연장 목록 조회 비로그인 사용자 구분 #89

ujiiin committed:
Merge pull request #92 from UMC-ShowHoo/feature/#89

[FEAT] 공연장 목록 조회 비로그인 사용자 구분 #89

dldusgh318 committed:
[FIX] dev pull (#68)

dldusgh318 committed:
[FIX] rentalFile에 show테이블 연결 수정 (#68)

hyoinkang committed:
feature : DB delete

hyoinkang committed:
Merge branch 'dev' of https://github.com/UMC-ShowHoo/ShowHoo-Server into feature/#91

dldusgh318 committed:
[FIX] dev merge (#68)

hyoinkang committed:
feature : 취소 가능 여부 로직 구현

hyoinkang committed:
Merge pull request #93 from UMC-ShowHoo/feature/#91

[FEAT] 회원 탈퇴 API 완성

hyoinkang committed:
Merge pull request #94 from UMC-ShowHoo/feature/#68

[FIX] rentalFile에 show테이블 연결 수정 (#68)

hyoinkang committed:
Merge pull request #94 from UMC-ShowHoo/feature/#68

[FIX] rentalFile에 show테이블 연결 수정 (#68)

hyoinkang committed:
modify : 관람 완료 조회 API 취소 내역 조회 API로 변경

hyoinkang committed:
Merge branch 'dev' of https://github.com/UMC-ShowHoo/ShowHoo-Server into feature/#95

ujiiin committed:
[REFACTOR] 파일 구조 리팩터링

hyoinkang committed:
feat : Shows 엔티티 isComplete init

ujiiin committed:
Merge remote-tracking branch 'origin/dev' into dev

hyoinkang committed:
Merge branch 'dev' of https://github.com/UMC-ShowHoo/ShowHoo-Server into feature/#95

hyoinkang committed:
refactor : isCancellable 실행 조건 추가

ujiiin committed:
[REFACTOR] space 엔티티 column nullable 설정 #96

ujiiin committed:
[REFACTOR] space 검색 필터링 API location 필터 수정 및 2개로 나눈 api 1개로 병합 #96

ujiiin committed:
[REFACTOR] spacePrefer 에러 핸들링 추가 #96

ujiiin committed:
Merge pull request #98 from UMC-ShowHoo/refactor/#96

[REFACTOR] Space, SpacePrefer 엔티티 설정 수정 및 에러 핸들링 추가

dldusgh318 committed:
[FEAT] 공연자 공연준비 : 취소 기한 API 추가 (#99)

dldusgh318 committed:
[MODIFY] 공연자 공연준비 : 포스터 s3 저장 (#101)

hyoinkang committed:
Merge pull request #97 from UMC-ShowHoo/feature/#95

[FEAT] 공연 취소 관련 로직 추가

ujiiin committed:
[FIX] 공연장 전체 조회 오류 수정

hyoinkang committed:
fix : 카카오 로그인 CORS 에러 수정

ujiiin committed:
[FIX] 에러 더블 체크 수정

hyoinkang committed:
Merge pull request #107 from UMC-ShowHoo/fix/#105

[FIX] 카카오 로그인 CORS 에러 수정

hyoinkang committed:
fix : 카카오 로그인 redirect uri 수정

dldusgh318 committed:
[FIX] rentalFIle 스웨거 오류 수정중 (#101)

dldusgh318 committed:
[FIX] dev pull (#101)

dldusgh318 committed:
[FIX] rentalFile 스웨거오류 수정 (#101)

dldusgh318 committed:
Merge branch 'dev' into feature/#99

dldusgh318 committed:
Merge pull request #100 from UMC-ShowHoo/feature/#99

[FEAT] 공연자 공연준비 : 취소 기한 API 추가 (#99)

dldusgh318 committed:
Merge branch 'dev' into feature/#101

dldusgh318 committed:
Merge pull request #103 from UMC-ShowHoo/feature/#101

[MODIFY] 공연자 공연준비 : 포스터 s3 저장 (#101)

ujiiin committed:
[FEAT] 공연자의 공연장 찜 유무 조회 #111 (#112)

* [FIX] space->spaces 엔드포인트 변경 #111

* [FEAT] 공연자의 공연장 찜 유무 조회 API 구현 #111

qormoon committed:
chore: 프로필 조회 시 이름, 전화번호 추가

qormoon committed:
chore: 공연장 등록 시 공연장 유형 추가 #4

qormoon committed:
chore: 스웨거 설명 추가 #4

hyesuhan committed:
fix : 공연장 대관 불가능 날짜 조회 API (#108)

* feat: 대관 가능 일정 확인

대관 가능 일정 확인 API

* fix: 공연 정보 작성 및 요청 사항

* fix: 공연 불가능 날짜 조회 API

* Revert "Merge branch 'dev' into feature/#102"

This reverts commit ca27162c89ec56669d2b41397a1bb2d3787521be, reversing
changes made to 8fba8ecb8bee1aac58aa94cb55e09a2857c49a48.

hyoinkang committed:
Revert "fix : 공연장 대관 불가능 날짜 조회 API (#108)" (#114)

This reverts commit 2a5c090d48f7ab32733780e9ce756dc4ede0b84a.

hyoinkang committed:
refactor : 조회 API 수정 (#110)

* modify : 공연 찜 리스트 조회 응답에 isComplete 추가

* modify : 공연 상세 조회 API description 관련 validation 추가

* fix : revert merge

hyoinkang committed:
fix : ddl-auto update로 수정

hyoinkang commented:
OAuth2와 Jwt 제외하고 코드 리뷰 마쳤습니다.
PR #7 , PR #15 , PR #40 , PR #47 페이지에 코멘트 달아놓았으니 확인해주세요

ujiiin committed:
[FIX] 공연자 마이페이지 조회 한개만 조회되도록 수정

ujiiin committed:
[FIX] performerProfileResponseDTO.profileDTO createdAt 추가

ujiiin committed:
[FIX] BaseEntity create_at -> createAt 변경

dldusgh318 committed:
[FEAT] 공연자 공연준비 - 포스터url 반환 API (#118) (#119)

* [FEAT] 공연자 공연준비 - 포스터 등록시 url 바로 반환 (#118)

* [MODIFY] 공연자 공연준비 - 공연정보 등록시 포스터를 멀티파트형식으로 받음 (#118)

* [FIX] 공연자 공연준비 - 포스터 url반환 api에서 performerProfileId 삭제 (#118)

* [FIX] 공연자 공연준비 - 포스터url API 스웨거 설명 수정 (#118)

hyesuhan committed:
fix: 대관 가능 날짜 확인 API

hyesuhan committed:
fix : 대관 가능 날싸 확인 API를 GETMAPPING (#121)

* feat: 대관 가능 일정 확인

대관 가능 일정 확인 API

* fix: 공연 정보 작성 및 요청 사항

* fix: 공연 불가능 날짜 조회 API

* Revert "Merge branch 'dev' into feature/#102"

This reverts commit ca27162c89ec56669d2b41397a1bb2d3787521be, reversing
changes made to 8fba8ecb8bee1aac58aa94cb55e09a2857c49a48.

* fix: 대관 가능 날짜 확인 API
patchmapping에서 getmapping으로

hyesuhan committed:
Revert "fix : 대관 가능 날싸 확인 API를 GETMAPPING (#121)"

This reverts commit 1ea22af228929347becf0e8b0aa968ed7907fcce.

hyesuhan committed:
ifs: 대관 날짜 list오류 수정 (#123)

* Revert "fix : 대관 가능 날싸 확인 API를 GETMAPPING (#121)"

This reverts commit 1ea22af228929347becf0e8b0aa968ed7907fcce.

* fix: 대관 가능 날짜 list 오류 수정

hyesuhan committed:
Merge branch 'dev' of https://github.com/UMC-ShowHoo/ShowHoo-Server into dev

hyesuhan committed:
공연장의 날짜별로 대관 내역 조회 API (#126)

* fix: 대관 가능 날짜 list 오류 수정

* [feat] 각 대관장 별 날짜 눌러서 대관 내역 조회하기

dldusgh318 committed:
[REFACTOR] 코드리뷰 수정사항 (#125)

* [FIX] dev pull

* [REFACTOR] 요청사항 200자 제한

* [REFACTOR] 공연자/공연장 - 큐시트작성 : 파일 미등록시, 에러 핸들링

ujiiin commented on src/main/java/umc/ShowHoo/web/performerProfile/service/PerformerProfileService.java:
```diff
@@ -0,0 +1,185 @@
+
+    public void updateProfileText(Long performerUserId, Long profileId, PerformerProfileRequestDTO.UpdateProfileTextDTO dto) {
+        Performer performer = performerRepository.findById(performerUserId)
+                .orElseThrow(() -> new IllegalArgumentException("Performer not found"));
```
error status 형식에 맞게 에러 핸들링해주세요(아래 다른 에러 처리들도)

ujiiin commented on src/main/java/umc/ShowHoo/web/performerProfile/dto/PerformerProfileRequestDTO.java:
```diff
@@ -0,0 +1,63 @@
+        private String name;
+        private String introduction;
+        private String phoneNumber;
+        private List<String> profileImageUrls;
```
profileImageUrl이 들어가는 List인데 List<URL>이 아닌 List<String>으로 설정한 이유가 있을까요?

ujiiin commented on src/main/java/umc/ShowHoo/web/performerProfile/service/PerformerProfileService.java:
```diff
@@ -0,0 +1,185 @@
+    private final UuidRepository uuidRepository;
+    private final ProfileImageRepository profileImageRepository;
+
+    public PerformerProfile createProfile(Long performerUserId, PerformerProfileRequestDTO.CreateProfileDTO profileDTO) {
```
@Transactional 어노테이션 추가해주세요

ujiiin commented on src/main/java/umc/ShowHoo/web/performerProfile/dto/PerformerProfileRequestDTO.java:
```diff
@@ -0,0 +1,63 @@
+    @AllArgsConstructor
+    public static class CreateProfileDTO{
+        //소속, 소개, 프로필 사진, 이름, 전번 추가
+        private String team;
```
notnull 등 validate 체크 해주세요

ujiiin commented on src/main/java/umc/ShowHoo/web/performerProfile/controller/PerformerProfileController.java:
```diff
@@ -0,0 +1,128 @@
+    @ApiResponses({
+            @io.swagger.v3.oas.annotations.responses.ApiResponse(responseCode = "200", description = "OK, 성공"),
+    })
+    public ApiResponse<List<String>>uploadProfileImages(@RequestPart List<MultipartFile> profileImages){
```
공연자 프로필 이미지 추가 API와 다른점이 뭔가요? 이미지 추가 API는 multipart file을 받고 s3에 저장 후 DB preofileImage 테이블에 url을 저장하는데, s3에 저장만 하는 API가 따로 있는 이유가 있을까요?

ujiiin reviewed (commented):
공연자 프로필 관련 
담당: 백지현
리뷰어: 변유진

ujiiin commented on src/main/java/umc/ShowHoo/web/space/dto/SpaceRequestDTO.java:
```diff
@@ -0,0 +1,88 @@
+    @NoArgsConstructor
+    @AllArgsConstructor
+    public static class SpaceRegisterRequestDTO {
+        private String name;
```
validate체크 해주세요

ujiiin commented on src/main/java/umc/ShowHoo/web/space/controller/SpaceController.java:
```diff
@@ -0,0 +1,173 @@
+            @RequestPart(required = false) MultipartFile spaceSeat) {
+        try {
+            // spaceUserId로 SpaceUser 조회
+            Optional<SpaceUser> optionalSpaceUser = spaceUserRepository.findById(spaceUserId);
```
spaceUser 존재 유무에 따른 에러핸들링은 service단에서 하는게 좋을 것 같아요

ujiiin reviewed (commented):
공연장 등록 관련 리뷰
담당: 백지현
리뷰어: 변유진

ujiiin commented on src/main/java/umc/ShowHoo/web/spaceApply/service/SpaceApplyService.java:
```diff
@@ -0,0 +1,136 @@
+    @Transactional
+    public void setSpaceApply(Long spaceId, Long spaceApplyId) {
+        Space space = spaceRepository.findById(spaceId)
+                .orElseThrow(() -> new SpaceApplyHandler(ErrorStatus.SPACE_NOT_FOUND));
```
Space에 대한 에러 핸들링을 할 때는 SpaceHandler로 에러 처리 해주세요

ujiiin commented on src/main/java/umc/ShowHoo/web/spaceApply/service/SpaceApplyService.java:
```diff
@@ -0,0 +1,136 @@
+                .orElseThrow(() -> new SpaceApplyHandler(ErrorStatus.SPACE_NOT_FOUND));
+
+        SpaceApply spaceApply = spaceApplyRepository.findById(spaceApplyId)
+                .orElseThrow(() -> new SpaceApplyHandler(ErrorStatus.SPACE_NOT_FOUND));
```
SpaceApply 존재 유무에 대한 에러 핸들링이니 ErrorStatus에 SPACE_NOT_FOUND가 아닌 spaceapply not found 에러를 생성해 사용해주세요

ujiiin commented on src/main/java/umc/ShowHoo/web/spaceApply/service/SpaceApplyService.java:
```diff
@@ -0,0 +1,136 @@
+    @Transactional
+    public List<Object> getSpaceApplyInfo(Long spaceId) {
+        Space space = spaceRepository.findById(spaceId)
+                .orElseThrow(() -> new SpaceApplyHandler(ErrorStatus.SPACE_NOT_FOUND));
```
spaceHandler로 에러 처리 해주세요

ujiiin commented on src/main/java/umc/ShowHoo/web/spaceApply/service/SpaceApplyService.java:
```diff
@@ -0,0 +1,136 @@
+        Space space = spaceRepository.findById(spaceId)
+                .orElseThrow(() -> new SpaceApplyHandler(ErrorStatus.SPACE_NOT_FOUND));
+
+        List<Object> spaceInfo = new ArrayList<>();
```
타입을 Object가 아닌 DTO로 명시해주세요.
여기에서는 List<spaceApply> 와 List<HolidayDTO>를 가지고 있는 DTO를 만들어야할 것 같네요

ujiiin commented on src/main/java/umc/ShowHoo/web/spaceApply/service/SpaceApplyService.java:
```diff
@@ -0,0 +1,136 @@
+
+    @Transactional(readOnly = true)
+    public SpaceApplyResponseDTO.SpaceApplyDetailDTO getSpaceApplyDetailsByDate(Long spaceId, LocalDate date) {
+        Optional<SpaceApply> spaceApplyOpt = spaceApplyRepository.findBySpaceIdAndDate(spaceId, date);
```
spaceId에 해당하는 Space가 존재하는지 에러핸들링 먼저 해주세요.

ujiiin reviewed (commented):
대관신청 승인/거절, 조회 관련
담당: 한혜수
리뷰어: 변유진

hyoinkang committed:
feat : 멤버 정보 조회 API 구현 (#117)

hyesuhan committed:
[feat]각 공연자 별 추가 장비 receipt & [fix]날짜로 대관 확인 시 performer poster과 타이틀 같이 (#128)

* fix: 대관 가능 날짜 list 오류 수정

* [feat] 각 대관장 별 날짜 눌러서 대관 내역 조회하기

* [feat] 각 공연자 별 추가 장비 receipt API, [fix] 공연장이 날짜를 통해 대관 수락 창 혹은 관련된 내용 뜨기 + title과 show photo로

* [feat] 각 공연자 별 추가 장비 receipt API, [fix] 공연장이 날짜를 통해 대관 수락 창 혹은 관련된 내용 뜨기 + title과 show photo로

ujiiin committed:
[FIX] spaceApply 취소 거절 구분

ujiiin committed:
Merge branch 'dev' of https://github.com/UMC-ShowHoo/ShowHoo-Server into dev

qormoon committed:
chore: 리뷰 엔티티 값 추가 #47

hyoinkang committed:
Revert "[feat]각 공연자 별 추가 장비 receipt & [fix]날짜로 대관 확인 시 performer poster과 타이틀 …" (#130)

This reverts commit fed674d01d096c6fc755397a396031d87d6ca381.

hyesuhan committed:
[fix] 공연장 영수증 API (#131)

* fix: 대관 가능 날짜 list 오류 수정

* [feat] 각 대관장 별 날짜 눌러서 대관 내역 조회하기

* [feat] 각 공연자 별 추가 장비 receipt API, [fix] 공연장이 날짜를 통해 대관 수락 창 혹은 관련된 내용 뜨기 + title과 show photo로

* [feat] 각 공연자 별 추가 장비 receipt API, [fix] 공연장이 날짜를 통해 대관 수락 창 혹은 관련된 내용 뜨기 + title과 show photo로

* [feat] 공연장 추가 서비스 영수증 발행(base fee + 추가 서비스 이름 및 가격 + total Price

dldusgh318 committed:
[REFACTOR] 코드 리뷰 수정사항 #124 (#132)

* [REFECTOR] 스웨거 설명 추가

* [FEAT] spaceApply, shows 일대일 연관관계 추가

* [FEAT] shows에 공연날짜, 디데이 관련 dto,converter 구현 (#124)

* [FEAT]공연 준비 시, 공연날짜 공연디데이 관련 로직 (#124)

* [FEAT]공연장 대여가 승인되지않은 경우 에러핸들링(#124)

* [FIX] 코드 정리 (#124)

* [FIX] 공연자 공연 준비 -공연 등록: rentalFile등록 시에 공연 id가 생성되므로, 공연 정보 set하는 방식으로 구현 (#124)

* [FIX] 공연장 공연 준비 - rentalFile 등록 시, 공연id 생성

* [FIX] rentalFile 연관관계 수정

* [FIX] 공연 포스터 오류 수정 (#124)

* [FIX] rentalFile spaceApply연관관계 삭제(이미 shows-spaceApply연관관계 있음)  (#124)

qormoon committed:
feat: 리뷰 작성 시간, 멤버 이름과 이미지 저장 #47

qormoon committed:
chore: 리뷰 답변 엔티티에 작성시간 추가 #47

qormoon committed:
feat: 리뷰 조회 api 반환값 추가 #47

qormoon committed:
feat: 리뷰 api 반환값 추가 #47

qormoon committed:
Merge branch 'dev' of https://github.com/UMC-ShowHoo/ShowHoo-Server into dev

qormoon committed:
chore: db 초기화

qormoon committed:
chore: db update 변경

hyoinkang committed:
[REFACTOR] 코드 리뷰 수정 사항 수정 (#134)

* refactor : 회원 탈퇴 requestBody 수정

* refactor : 코드 리뷰 이후 자잘한 수정 사항 수정

* refactor : 공연 상세 조회 API에 장소 정보 추가

* Update BookConverter ! 수정

.

hyoinkang committed:
chore : authorization 수정

hyoinkang committed:
Merge branch 'dev' of https://github.com/UMC-ShowHoo/ShowHoo-Server into dev

qormoon committed:
[REFACTOR] 공연장 API 수정 (#136)

* feat: 공연장 세부정보 API 반환값 추가 #133

* feat: 공연장 세부정보 헤더 조회 API #133

* chore: 리뷰 조회 api 수정 #133

hyoinkang committed:
[FEAT] 공연 준비 - 입장 관리 API 구현 (#137)

* feat : 입장관리 API 구현

* fix : 입장관리 API 수정

hyoinkang committed:
refactor : 공연 상세정보 조회 API response 수정

hyoinkang committed:
[FEAT] 공연 매진 여부 및 남은 티켓 조회 로직 추가 (#139)

* fix : 남은 티켓 매수 조회

* chore : 티켓 매진 시 낙관적 락 방식으로 처리

qormoon committed:
chore: 대관신청 시 spaceId로 변경 #133

qormoon committed:
Merge branch 'dev' of https://github.com/UMC-ShowHoo/ShowHoo-Server into dev

qormoon committed:
Revert "Merge branch 'dev' of https://github.com/UMC-ShowHoo/ShowHoo-Server into dev"

This reverts commit e13381c50b74e8b024840fac5d775a40a8dcac5e, reversing
changes made to d8b74c39f410589290c28fae1656e980cc0a0c05.

qormoon committed:
Reapply "Merge branch 'dev' of https://github.com/UMC-ShowHoo/ShowHoo-Server into dev"

This reverts commit 19a9add419220b1782b72d30f6177e013cbd94c0.

dldusgh318 committed:
[FEAT] 공연자 공연준비 - 예매자관리에서 예매자의 예매 상태 넘기기 (#124) (#141)

* [REFECTOR] 스웨거 설명 추가

* [FEAT] spaceApply, shows 일대일 연관관계 추가

* [FEAT] shows에 공연날짜, 디데이 관련 dto,converter 구현 (#124)

* [FEAT]공연 준비 시, 공연날짜 공연디데이 관련 로직 (#124)

* [FEAT]공연장 대여가 승인되지않은 경우 에러핸들링(#124)

* [FIX] 코드 정리 (#124)

* [FIX] 공연자 공연 준비 -공연 등록: rentalFile등록 시에 공연 id가 생성되므로, 공연 정보 set하는 방식으로 구현 (#124)

* [FIX] 공연장 공연 준비 - rentalFile 등록 시, 공연id 생성

* [FIX] rentalFile 연관관계 수정

* [FIX] 공연 포스터 오류 수정 (#124)

* [FIX] rentalFile spaceApply연관관계 삭제(이미 shows-spaceApply연관관계 있음)  (#124)

* [FEAT] 공연자 공연준비 - 예매자 예매 상태 넘기기 (#124)

ujiiin committed:
[FEAT] 찜 유무 조회 API에 spacePreferId 추가

ujiiin committed:
[FIX] 에러핸들링 수정

qormoon committed:
chore: 토큰없이 api 접근 허용

qormoon committed:
Merge branch 'dev' of https://github.com/UMC-ShowHoo/ShowHoo-Server into dev

qormoon committed:
fix: 시큐리티 경로 매칭 수정

hyesuhan committed:
[MODIFIY] 공연장 영수증 API - 날짜, 추가 서비스 (#143)

* feat: 대관 가능 일정 확인

대관 가능 일정 확인 API

* fix: 공연 정보 작성 및 요청 사항

* fix: 공연 불가능 날짜 조회 API

* Revert "Merge branch 'dev' into feature/#102"

This reverts commit ca27162c89ec56669d2b41397a1bb2d3787521be, reversing
changes made to 8fba8ecb8bee1aac58aa94cb55e09a2857c49a48.

* fix: 대관 가능 날짜 확인 API
patchmapping에서 getmapping으로

* chore: 리뷰 엔티티 값 추가 #47

* feat: 리뷰 작성 시간, 멤버 이름과 이미지 저장 #47

* chore: 리뷰 답변 엔티티에 작성시간 추가 #47

* feat: 리뷰 조회 api 반환값 추가 #47

* feat: 리뷰 api 반환값 추가 #47

* Revert "fix : 대관 가능 날싸 확인 API를 GETMAPPING (#121)"

This reverts commit 1ea22af228929347becf0e8b0aa968ed7907fcce.

* ifs: 대관 날짜 list오류 수정 (#123)

* Revert "fix : 대관 가능 날싸 확인 API를 GETMAPPING (#121)"

This reverts commit 1ea22af228929347becf0e8b0aa968ed7907fcce.

* fix: 대관 가능 날짜 list 오류 수정

* 공연장의 날짜별로 대관 내역 조회 API (#126)

* fix: 대관 가능 날짜 list 오류 수정

* [feat] 각 대관장 별 날짜 눌러서 대관 내역 조회하기

* [REFACTOR] 코드리뷰 수정사항 (#125)

* [FIX] dev pull

* [REFACTOR] 요청사항 200자 제한

* [REFACTOR] 공연자/공연장 - 큐시트작성 : 파일 미등록시, 에러 핸들링

* [FIX] spaceApply 취소 거절 구분

* feat : 멤버 정보 조회 API 구현 (#117)

* [feat]각 공연자 별 추가 장비 receipt & [fix]날짜로 대관 확인 시 performer poster과 타이틀 같이 (#128)

* fix: 대관 가능 날짜 list 오류 수정

* [feat] 각 대관장 별 날짜 눌러서 대관 내역 조회하기

* [feat] 각 공연자 별 추가 장비 receipt API, [fix] 공연장이 날짜를 통해 대관 수락 창 혹은 관련된 내용 뜨기 + title과 show photo로

* [feat] 각 공연자 별 추가 장비 receipt API, [fix] 공연장이 날짜를 통해 대관 수락 창 혹은 관련된 내용 뜨기 + title과 show photo로

* Revert "[feat]각 공연자 별 추가 장비 receipt & [fix]날짜로 대관 확인 시 performer poster과 타이틀 …" (#130)

This reverts commit fed674d01d096c6fc755397a396031d87d6ca381.

* [fix] 공연장 영수증 API (#131)

* fix: 대관 가능 날짜 list 오류 수정

* [feat] 각 대관장 별 날짜 눌러서 대관 내역 조회하기

* [feat] 각 공연자 별 추가 장비 receipt API, [fix] 공연장이 날짜를 통해 대관 수락 창 혹은 관련된 내용 뜨기 + title과 show photo로

* [feat] 각 공연자 별 추가 장비 receipt API, [fix] 공연장이 날짜를 통해 대관 수락 창 혹은 관련된 내용 뜨기 + title과 show photo로

* [feat] 공연장 추가 서비스 영수증 발행(base fee + 추가 서비스 이름 및 가격 + total Price

* [REFACTOR] 코드 리뷰 수정사항 #124 (#132)

* [REFECTOR] 스웨거 설명 추가

* [FEAT] spaceApply, shows 일대일 연관관계 추가

* [FEAT] shows에 공연날짜, 디데이 관련 dto,converter 구현 (#124)

* [FEAT]공연 준비 시, 공연날짜 공연디데이 관련 로직 (#124)

* [FEAT]공연장 대여가 승인되지않은 경우 에러핸들링(#124)

* [FIX] 코드 정리 (#124)

* [FIX] 공연자 공연 준비 -공연 등록: rentalFile등록 시에 공연 id가 생성되므로, 공연 정보 set하는 방식으로 구현 (#124)

* [FIX] 공연장 공연 준비 - rentalFile 등록 시, 공연id 생성

* [FIX] rentalFile 연관관계 수정

* [FIX] 공연 포스터 오류 수정 (#124)

* [FIX] rentalFile spaceApply연관관계 삭제(이미 shows-spaceApply연관관계 있음)  (#124)

* chore: db 초기화

* chore: db update 변경

* chore: 대관신청 시 spaceId로 변경 #133

* chore : authorization 수정

* [REFACTOR] 코드 리뷰 수정 사항 수정 (#134)

* refactor : 회원 탈퇴 requestBody 수정

* refactor : 코드 리뷰 이후 자잘한 수정 사항 수정

* refactor : 공연 상세 조회 API에 장소 정보 추가

* Update BookConverter ! 수정

.

* [REFACTOR] 공연장 API 수정 (#136)

* feat: 공연장 세부정보 API 반환값 추가 #133

* feat: 공연장 세부정보 헤더 조회 API #133

* chore: 리뷰 조회 api 수정 #133

* [FEAT] 공연 준비 - 입장 관리 API 구현 (#137)

* feat : 입장관리 API 구현

* fix : 입장관리 API 수정

* refactor : 공연 상세정보 조회 API response 수정

* [FEAT] 공연 매진 여부 및 남은 티켓 조회 로직 추가 (#139)

* fix : 남은 티켓 매수 조회

* chore : 티켓 매진 시 낙관적 락 방식으로 처리

* Revert "Merge branch 'dev' of https://github.com/UMC-ShowHoo/ShowHoo-Server into dev"

This reverts commit e13381c50b74e8b024840fac5d775a40a8dcac5e, reversing
changes made to d8b74c39f410589290c28fae1656e980cc0a0c05.

* Reapply "Merge branch 'dev' of https://github.com/UMC-ShowHoo/ShowHoo-Server into dev"

This reverts commit 19a9add419220b1782b72d30f6177e013cbd94c0.

* chore: 토큰없이 api 접근 허용

* [FEAT] 공연자 공연준비 - 예매자관리에서 예매자의 예매 상태 넘기기 (#124) (#141)

* [REFECTOR] 스웨거 설명 추가

* [FEAT] spaceApply, shows 일대일 연관관계 추가

* [FEAT] shows에 공연날짜, 디데이 관련 dto,converter 구현 (#124)

* [FEAT]공연 준비 시, 공연날짜 공연디데이 관련 로직 (#124)

* [FEAT]공연장 대여가 승인되지않은 경우 에러핸들링(#124)

* [FIX] 코드 정리 (#124)

* [FIX] 공연자 공연 준비 -공연 등록: rentalFile등록 시에 공연 id가 생성되므로, 공연 정보 set하는 방식으로 구현 (#124)

* [FIX] 공연장 공연 준비 - rentalFile 등록 시, 공연id 생성

* [FIX] rentalFile 연관관계 수정

* [FIX] 공연 포스터 오류 수정 (#124)

* [FIX] rentalFile spaceApply연관관계 삭제(이미 shows-spaceApply연관관계 있음)  (#124)

* [FEAT] 공연자 공연준비 - 예매자 예매 상태 넘기기 (#124)

* [FEAT] 찜 유무 조회 API에 spacePreferId 추가

* [FIX] 에러핸들링 수정

* fix: 시큐리티 경로 매칭 수정

* [MODIFIY] 공연장 영수증 서비스

---------

Co-authored-by: jihyeon baek <qormoon@sookmyung.ac.kr>
Co-authored-by: dldusgh318 <125371704+dldusgh318@users.noreply.github.com>
Co-authored-by: byj7176@gmail.com <byj7176@gmail.com>
Co-authored-by: hyoinkang <103008960+hyoinkang@users.noreply.github.com>
Co-authored-by: hyoinkang <hyoin1704@gmail.com>
Co-authored-by: jihyeon baek <137600913+qormoon@users.noreply.github.com>

hyesuhan committed:
[MODIFIY] 공연장 수락 슈팅 문제 (#145)

* fix: 대관 가능 날짜 list 오류 수정

* [feat] 각 대관장 별 날짜 눌러서 대관 내역 조회하기

* [feat] 각 공연자 별 추가 장비 receipt API, [fix] 공연장이 날짜를 통해 대관 수락 창 혹은 관련된 내용 뜨기 + title과 show photo로

* [feat] 각 공연자 별 추가 장비 receipt API, [fix] 공연장이 날짜를 통해 대관 수락 창 혹은 관련된 내용 뜨기 + title과 show photo로

* [feat] 공연장 추가 서비스 영수증 발행(base fee + 추가 서비스 이름 및 가격 + total Price

* [fix] 공연장 추가 서비스 영수증 발행(base fee + 추가 서비스 이름 및 가격 + total Price

* [MODIFIY] 공연장 영수증 서비스

* [MODIFIY] 공연장 영수증 서비스

* [MODIFIY] 공연장 영수증 서비스

hyoinkang committed:
chore : shows - showsDsc cascade 설정

hyesuhan committed:
[FIX] Fatch error 수정 (#146)

* fix: 대관 가능 날짜 list 오류 수정

* [feat] 각 대관장 별 날짜 눌러서 대관 내역 조회하기

* [feat] 각 공연자 별 추가 장비 receipt API, [fix] 공연장이 날짜를 통해 대관 수락 창 혹은 관련된 내용 뜨기 + title과 show photo로

* [feat] 각 공연자 별 추가 장비 receipt API, [fix] 공연장이 날짜를 통해 대관 수락 창 혹은 관련된 내용 뜨기 + title과 show photo로

* [feat] 공연장 추가 서비스 영수증 발행(base fee + 추가 서비스 이름 및 가격 + total Price

* [fix] 공연장 추가 서비스 영수증 발행(base fee + 추가 서비스 이름 및 가격 + total Price

* [MODIFIY] 공연장 영수증 서비스

* [MODIFIY] 공연장 영수증 서비스

* [MODIFIY] 공연장 영수증 서비스

* [FIX] Fatch error security 수정

qormoon committed:
chore: 오류 주석 처리

qormoon committed:
Merge branch 'dev' of https://github.com/UMC-ShowHoo/ShowHoo-Server into dev

dldusgh318 committed:
[FIX] 공연장 공연준비 - 큐시트 작성API 엔드포인트 수정 (#124) (#147)

* [FIX] 공연장 공연준비 - 큐시트 작성API 엔드포인트 수정 (#124)

hyoinkang committed:
[MODIFY] 공연 조회 쿼리 변경 (#149)

* modify : shows - 티켓 발행 여부 확인 로직 구현

* fix : 공연 조회 쿼리 변경

ujiiin committed:
[FIX] deleteSpaceApply 에러 수정

ujiiin committed:
[FIX] createSpaceApply notification 에러 수정

ujiiin committed:
[FIX] 공연장 전체조회, 검색 API에서 preferred 제외, spaceId 추가

ujiiin committed:
[FIX] notification 오타 수정

hyoinkang committed:
chore : 공연게시글 검색 조회 엔드포인트 변경

ujiiin committed:
[FIX] spaceApply, shows 연관관계 수정

ujiiin committed:
Merge branch 'dev' of https://github.com/UMC-ShowHoo/ShowHoo-Server into dev

ujiiin committed:
[FIX] spacePrefer API swagger에 token 명시

hyoinkang committed:
[MODIFY] 공연게시글 조회 방식 변경 (#152)

* modify : 공연 게시글 조회 방식 변경

* modify : 공연 게시글 조회 dto 변경

hyoinkang committed:
chore : 엔드포인트 수정

ujiiin committed:
Merge branch 'dev' of https://github.com/UMC-ShowHoo/ShowHoo-Server into dev

ujiiin committed:
Merge branch 'dev' of https://github.com/UMC-ShowHoo/ShowHoo-Server into dev

hyoinkang committed:
chore : /aud/search/* 토큰 없이 API 접근 허용

ujiiin committed:
[FIX] spaceApply 삭제 오류 해결

dldusgh318 committed:
[REFACTOR] 공연자 공연준비 - 큐시트 작성: spaceApplyId로 shows 큐시트파일 저장 (#124) (#153)


* REFACTOR] 공연자 공연준비 - 큐시트 작성: spaceApplyId로 shows 큐시트파일 저장 (#124)

hyesuhan committed:
[FIX] 대관 날짜로 공연장 확인 및 shows 확인 (#154)

qormoon committed:
chore: space테이블에 위도 경도 추가

dldusgh318 committed:
[FIX] shows-spaceApply cascade 수정, shows-rentalFile cascade 추가 (#155)

* [REFECTOR] 스웨거 설명 추가

* [FEAT] spaceApply, shows 일대일 연관관계 추가

* [FEAT] shows에 공연날짜, 디데이 관련 dto,converter 구현 (#124)

* [FEAT]공연 준비 시, 공연날짜 공연디데이 관련 로직 (#124)

* [FEAT]공연장 대여가 승인되지않은 경우 에러핸들링(#124)

* [FIX] 코드 정리 (#124)

* [FIX] 공연자 공연 준비 -공연 등록: rentalFile등록 시에 공연 id가 생성되므로, 공연 정보 set하는 방식으로 구현 (#124)

* [FIX] 공연장 공연 준비 - rentalFile 등록 시, 공연id 생성

* [FIX] rentalFile 연관관계 수정

* [FIX] 공연 포스터 오류 수정 (#124)

* [FIX] rentalFile spaceApply연관관계 삭제(이미 shows-spaceApply연관관계 있음)  (#124)

* [FEAT] 공연자 공연준비 - 예매자 예매 상태 넘기기 (#124)

* [FIX] 공연장 공연준비 - 큐시트 작성API 엔드포인트 수정 (#124)

* REFACTOR] 공연자 공연준비 - 큐시트 작성: spaceApplyId로 shows 큐시트파일 저장 (#124)

dldusgh318 committed:
Feature/#124 (#156)

* [REFECTOR] 스웨거 설명 추가

* [FEAT] spaceApply, shows 일대일 연관관계 추가

* [FEAT] shows에 공연날짜, 디데이 관련 dto,converter 구현 (#124)

* [FEAT]공연 준비 시, 공연날짜 공연디데이 관련 로직 (#124)

* [FEAT]공연장 대여가 승인되지않은 경우 에러핸들링(#124)

* [FIX] 코드 정리 (#124)

* [FIX] 공연자 공연 준비 -공연 등록: rentalFile등록 시에 공연 id가 생성되므로, 공연 정보 set하는 방식으로 구현 (#124)

* [FIX] 공연장 공연 준비 - rentalFile 등록 시, 공연id 생성

* [FIX] rentalFile 연관관계 수정

* [FIX] 공연 포스터 오류 수정 (#124)

* [FIX] rentalFile spaceApply연관관계 삭제(이미 shows-spaceApply연관관계 있음)  (#124)

* [FEAT] 공연자 공연준비 - 예매자 예매 상태 넘기기 (#124)

* [FIX] 공연장 공연준비 - 큐시트 작성API 엔드포인트 수정 (#124)

* REFACTOR] 공연자 공연준비 - 큐시트 작성: spaceApplyId로 shows 큐시트파일 저장 (#124)

* [REFACTOR] performerProfileId 수정(#124)

ujiiin committed:
[FIX] spaceApply, shows 연관관계 수정

ujiiin committed:
[FIX] spaceApply service 수정

dldusgh318 committed:
[REFACTOR] 공연자 공연준비 시 큐시트 폼 다운 엔드포인트 변경(#124) (#157)

* [REFECTOR] 스웨거 설명 추가

* [FEAT] spaceApply, shows 일대일 연관관계 추가

* [FEAT] shows에 공연날짜, 디데이 관련 dto,converter 구현 (#124)

* [FEAT]공연 준비 시, 공연날짜 공연디데이 관련 로직 (#124)

* [FEAT]공연장 대여가 승인되지않은 경우 에러핸들링(#124)

* [FIX] 코드 정리 (#124)

* [FIX] 공연자 공연 준비 -공연 등록: rentalFile등록 시에 공연 id가 생성되므로, 공연 정보 set하는 방식으로 구현 (#124)

* [FIX] 공연장 공연 준비 - rentalFile 등록 시, 공연id 생성

* [FIX] rentalFile 연관관계 수정

* [FIX] 공연 포스터 오류 수정 (#124)

* [FIX] rentalFile spaceApply연관관계 삭제(이미 shows-spaceApply연관관계 있음)  (#124)

* [FEAT] 공연자 공연준비 - 예매자 예매 상태 넘기기 (#124)

* [FIX] 공연장 공연준비 - 큐시트 작성API 엔드포인트 수정 (#124)

* REFACTOR] 공연자 공연준비 - 큐시트 작성: spaceApplyId로 shows 큐시트파일 저장 (#124)

* [REFACTOR] performerProfileId 수정(#124)

* [REFACTOR] 공연자 공연준비 시 큐시트 폼 다운 엔드포인트 변경(#124)

qormoon committed:
chore: space테이블에 위도 경도 추가

ujiiin committed:
Revert "[FIX] spaceApply service 수정"

This reverts commit fc185a877a96f151f96cd21f5fd95efd954f23bd.

ujiiin committed:
Revert "[FIX] spaceApply, shows 연관관계 수정"

This reverts commit dc9c3470f5429b49c9c2133de4cb26ab7ad2dfb5.

ujiiin committed:
Merge branch 'dev' of https://github.com/UMC-ShowHoo/ShowHoo-Server into dev

ujiiin committed:
[FIX] spaceApply, shows 연관관계 수정

hyoinkang committed:
modify : 입장 관리 API 조회 방식 변경 (#159)

ujiiin committed:
[FIX] DB create

ujiiin committed:
Merge branch 'dev' of https://github.com/UMC-ShowHoo/ShowHoo-Server into dev

ujiiin committed:
[FIX] DB update

hyoinkang committed:
chore : redirectUri 변경

ujiiin committed:
[FIX] yml 설정

ujiiin committed:
Merge branch 'dev' of https://github.com/UMC-ShowHoo/ShowHoo-Server into dev

ujiiin committed:
[FIX] yml 설정

hyoinkang committed:
chore : redirectUri 변경

hyoinkang committed:
Merge branch 'dev' of https://github.com/UMC-ShowHoo/ShowHoo-Server into dev

ujiiin committed:
[ci] ci/cd오류

ujiiin committed:
Merge branch 'dev' of https://github.com/UMC-ShowHoo/ShowHoo-Server into dev

ujiiin committed:
redirect-uri 변경

hyoinkang committed:
chore : redirectUri 변경

dldusgh318 committed:
[FEAT] 전체 예매자 조회  (#160)

* [REFECTOR] 스웨거 설명 추가

* [FEAT] spaceApply, shows 일대일 연관관계 추가

* [FEAT] shows에 공연날짜, 디데이 관련 dto,converter 구현 (#124)

* [FEAT]공연 준비 시, 공연날짜 공연디데이 관련 로직 (#124)

* [FEAT]공연장 대여가 승인되지않은 경우 에러핸들링(#124)

* [FIX] 코드 정리 (#124)

* [FIX] 공연자 공연 준비 -공연 등록: rentalFile등록 시에 공연 id가 생성되므로, 공연 정보 set하는 방식으로 구현 (#124)

* [FIX] 공연장 공연 준비 - rentalFile 등록 시, 공연id 생성

* [FIX] rentalFile 연관관계 수정

* [FIX] 공연 포스터 오류 수정 (#124)

* [FIX] rentalFile spaceApply연관관계 삭제(이미 shows-spaceApply연관관계 있음)  (#124)

* [FEAT] 공연자 공연준비 - 예매자 예매 상태 넘기기 (#124)

* [FIX] 공연장 공연준비 - 큐시트 작성API 엔드포인트 수정 (#124)

* REFACTOR] 공연자 공연준비 - 큐시트 작성: spaceApplyId로 shows 큐시트파일 저장 (#124)

* [REFACTOR] performerProfileId 수정(#124)

* [REFACTOR] 공연자 공연준비 시 큐시트 폼 다운 엔드포인트 변경(#124)

* 전체 예매자 조회 (#124)

hyoinkang committed:
chore : 위도 경도 추가

hyoinkang committed:
chore : redirectUri 변경

ujiiin committed:
[FIX] shows, spaceApply 연관관계 수정

ujiiin committed:
Merge branch 'dev' of https://github.com/UMC-ShowHoo/ShowHoo-Server into dev

ujiiin committed:
[FIX] shows, spaceApply 연관관계 수정

hyesuhan committed:
[ifx] (#161)

* [FIX] 대관 날짜로 공연장 확인 및 shows 확인

* [FIX] 대관 날짜로 공연장 확인 및 shows 확인
